www/_template/guides/svelte.md

```diff
@@ -0,0 +1,309 @@
+
+```diff
+// snowpack.config.js
+
+  mount: {
+-   /* ... */
++   // directory name: 'build directory'
++   public: '/',
++   src: '/_dist_',
+  },
+```
+
+<img src="/img/guides/react/folderstructure.png" alt="The original file configuration had Snowpack building the directory structure the same as the directories in the project, including root. Now the configuration builds only src and public. Src to the dist folder and public to root." />
+
+`mount` is part of the [Snowpack Configuration API](https://www.snowpack.dev/#configuration). It allows you to customize the file structure of your project. The key is the name of the directory and the value is where you'd like them in the final build. With this new configuration, Snowpack builds files in `public` like `public/index.css` directory into `index.css`. It builds files in `src` like `src/index.js` into `/_dist_/index.js`, so change that path in your `index.html`:
+
+```diff
+<!-- public/index.html -->
+
+  <body>
+-   <h1>Welcome to Snowpack!</h1>
+-   <script type="module" src="/index.js"></script>
++   <script type="module" src="/_dist_/index.js"></script>
+  </body>
+```
+
+You'll need to restart Snowpack for configuration file changes. When you start up again, if it worked, it should look the same.
+
+## Adding an animated Svelte Logo
+
+In Svelte you can add CSS directly to your component. This step demonstrates this capability by adding an animated logo.
+
+Download [`logo.svg`](https://github.com/snowpackjs/snowpack/blob/master/create-snowpack-app/app-template-svelte/public/logo.svg) to your `public` directory. Now you can add it to your `App.svelte`
+
+```diff
+<!-- src/App.svelte -->
+
+    <header class="App-header">
++       <img src="/logo.svg" class="App-logo" alt="logo" />
+        <a class="App-link" href="https://svelte.dev" target="_blank" rel="noopener noreferrer">
+          Learn Svelte
+```
+
+Image: Site showing the new logo
+
+With Svelte, CSS can go directly in your `.svelte` component. Add this code to the top of `App.svelte` between the `<style>` tags:
+
+```html
+<!-- src/App.svelte -->
+
+<style>
+  .App-header {
+    background-color: #f9f6f6;
+    color: #333;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+  }
+  .App-logo {
+    height: 36vmin;
+    pointer-events: none;
+    margin-bottom: 3rem;
+    animation: App-logo-spin infinite 1.6s ease-in-out alternate;
+  }
+  @keyframes App-logo-spin {
+    from {
+      transform: scale(1);
+    }
+    to {
+      transform: scale(1.06);
+    }
+  }
+</style>
+```
+
+> 💡 Tip: Snowpack has built-in support for [CSS Modules](https://www.snowpack.dev/#import-css-modules) and if you'd like to use Sass there is an official [Sass Plugin](https://www.snowpack.dev/#sass).
```

Comment:
I’m confused by this tip: as far as I can see, you cannot use CSS modules in a Svelte/Snowpack app (realise I’m new to both). e.g., if you were to `import styles from 'style.module.css'` in the script section. You cannot use them in the Svelte component’s HTML. (Unless I’m holding it wrong.) Again, as far as I can see, the Svelte, `style` tag serves the same purpose, albeit inline. A quick web search didn’t reveal any easy way to include external CSS in a Svelte component. I’m assuming you’re talking about CSS module support in general in Snowpack here and not in Svelte when used with Snowpack but I guess you can see how it can be confusing in the current context. Again, I might be missing something.

__Update:__ this isn’t CSS modules but if you want to keep your CSS external to your Svelte component, it looks like you _can_ use an `@import` __if__ you set `lang='css'` on your style tag.

e.g., Move the CSS in the example into a file called `App.css` in the same folder as `App.svelte`, then modify the latter to import it:

```html
<style lang='css'>
  @import url('/_dist_/App.css')
</style>
```

Not sure if this is documented anywhere else, encouraged/discouraged, or whether this is the best place to mention but still, good to know.

Reply:
Good catch, this doesn't make much sense in a Svelte guide